genesyscloud/util_retries.go

```diff
@@ -141,3 +141,14 @@ func GetBody(apiResponse *platformclientv2.APIResponse) string {
 	}
 	return ""
 }
+
+func IsStatus409(resp *platformclientv2.APIResponse, additionalCodes ...int) bool {
```

Comment:
can you add a unit test for this

Reply:
Just to confirm because I don't see unit tests for similar function IsStatus400 or IsStatus404, do you want the unit test for function IsStatus409?